### DISCUSSION TITLE
Multiple pictures with random seed

### DISCUSSION THREAD

**Discussion by realm174** [lllyasviel/Fooocus#1602] [Q&A]
Hello, I'm fairly new to this whole thing but I'm having a blast and it's beginning to make sense.  

When generating multiple pictures, the seed for each picture is incremented by one.  Is there a way to, instead, have it random for each picture instead of an incremental of the previous one?

**Comment by stephantual** [comment]
yeah in an ideal world, given the HIGHLY random nature of results even on img2img with detailed prompts, we woudl be able to 'batch' things incrementally (say a range of weigh on any given sliders). Sadly I do not know how to do this :(
